### PR TITLE
[ioctl] Dev account verify

### LINKED ISSUE #3003: [ioctl] build `account verify` command line into new ioctl
The new ioctl (./ioctl/newcmd) is a refactored version of the old ioctl(./ioctl/cmd). It includes two major changes:
- The [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) is passed when constructing the Cobra command.
- Unit tests are to be added for each command.

The newcmd code currently is a draft. The code of old ioctl can be used for reference.

### What would you like to be added
add `accountverify.go` and `accountverify_test.go` in `./ioctl/newcmd/account`. 
**Code for reference**:
https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/account/accountverify.go

### PR DESCRIPTION
**Pull request by LuckyPigeon**
fix #3003

### CODE REVIEW

File: ioctl/newcmd/account/accountverify.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			addr := priKey.PublicKey().Address()
+			if addr == nil {
+				return output.NewError(output.ConvertError, "failed to convert public key into address", nil)
+			}
+			message := verifyMessage{
+				Address:   addr.String(),
+				PublicKey: fmt.Sprintf("%x", priKey.PublicKey().Bytes()),
+			}
+			priKey.Zero()
+			fmt.Println(message.String())
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountverify.go:61]: client.Print

File: ioctl/newcmd/account/accountverify.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	return cmd
+}
+
+type verifyMessage struct {
+	Address   string `json:"address"`
+	PublicKey string `json:"publicKey"`
+}
+
+func (m *verifyMessage) String() string {
+	if output.Format == "" {
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountverify.go:74]: @huof6829 TODO: Hide output.Format in some way

File: ioctl/newcmd/account/accountverify_test.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	defer func() {
+		require.NoError(t, os.RemoveAll(testAccountFolder))
+	}()
+
+	t.Run("verify account successfully", func(t *testing.T) {
+		RawPrivateKey := "cfa6ef757dee2e50351620dca002d32b9c090cfda55fb81f37f1d26b273743f1"
+
+		cmd := NewAccountVerify(client)
+		result, err := util.ExecuteCmd(cmd, RawPrivateKey)
+		fmt.Println("Result: ", result)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountverify_test.go:40]: validate result in some way or just ignore it.

**Review by CoderZhi — COMMENTED** [review]
@huof6829 please review this PR

File: ioctl/newcmd/account/accountverify.go (hunk 1 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountverify.go:61]: Do you mean `client.PrintInfo` in https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go ?

**Comment by huof6829** [comment]
add sub command in NewAccountCmd of account.go

File: ioctl/newcmd/account/accountverify.go [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+			client.PrintInfo(message.String())
+			return nil
+		},
+	}
+}
+
+type verifyMessage struct {
+	Address   string `json:"address"`
+	PublicKey string `json:"publicKey"`
+}
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountverify.go:88]: we may delete this struct.

File: ioctl/newcmd/account/accountverify_test.go [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+func TestNewAccountVerify(t *testing.T) {
+	require := require.New(t)
+	ctrl := gomock.NewController(t)
+	client := mock_ioctlclient.NewMockClient(ctrl)
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslationString", config.English).AnyTimes()
+
+	t.Run("verify account successfully", func(t *testing.T) {
+		client.EXPECT().PrintInfo(gomock.Any()).Do(func(_ string) {
+			fmt.Println("enter private key:")
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountverify_test.go:30]: replaced with `require.Equal("enter private key:", input)`.

File: ioctl/newcmd/account/accountverify_test.go [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+		})
+		client.EXPECT().ReadSecret().Return("cfa6ef757dee2e50351620dca002d32b9c090cfda55fb81f37f1d26b273743f1", nil)
+		client.EXPECT().PrintInfo(gomock.Any()).Do(func(info string) {
+			fmt.Println(info)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountverify_test.go:34]: delete or verify info. Don't print it.

File: ioctl/newcmd/account/accountverify_test.go [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+		})
+
+		cmd := NewAccountVerify(client)
+		_, err := util.ExecuteCmd(cmd)
+		require.NoError(err)
+	})
+
+	t.Run("failed to covert hex string to private key", func(t *testing.T) {
+		client.EXPECT().PrintInfo(gomock.Any()).Do(func(_ string) {
+			fmt.Println("enter private key:")
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountverify_test.go:44]: same here.

**Review by CoderZhi — APPROVED** [review]

File: ioctl/newcmd/account/accountverify_test.go (hunk 5 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountverify_test.go:30]: This line is simulating the `client.PrintInfo` in `accountverify.go` and print the hint message. 
So this line can't use `require.Equal` nor be deleted.

File: ioctl/newcmd/account/accountverify_test.go (hunk 7 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountverify_test.go:44]: This line is simulating the client.PrintInfo in accountverify.go and print the hint message.
So this line can't use require.Equal nor be deleted.

File: ioctl/newcmd/account/accountverify.go (hunk 4 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountverify.go:88]: But what should we print at the end of command

File: ioctl/newcmd/account/accountverify_test.go (hunk 6 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountverify_test.go:34]: Just a simulation for `client.PrintInfo` in `accountverify.go` L79

**Review by huof6829 — APPROVED** [review]

**Pull request merged**